glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -316,7 +316,12 @@ glusterfs_handle_terminate(rpcsvc_request_t *req)
     xlator_list_t **trav_p = NULL;
     gf_boolean_t lockflag = _gf_false;
     gf_boolean_t still_bricks_attached = _gf_false;
+    dict_t *dict = NULL;
+    xlator_t *this = NULL;
+    char *value = NULL;
+    gf_boolean_t enabled = _gf_false;
```

Comment:
Maybe a more meaningful name would be better, like `graceful` or anything similar. `enabled` seems too generic to me and it's not clear what is enabled without code inspection.